decisions/0002-record-architecture-decisions.md

```diff
@@ -0,0 +1,49 @@
+# 2. Communicating NIST's OSCAL Events to the Public
+
+Date: 2023-02-03
+
+## Status
+
+Feedback Requested
+
+## Context
+
+The NIST OSCAL Team has necessary touchpoints with the community, among them structured and unstructured meetings with community stakeholders. Many of these meetings are events occuring at set frequencies, while others do not. In all cases, the community, past and present, has expressed the need for invitations to events that are easily accessible with the most current information possible.
+
+## Decision
+
+The NIST OSCAL Team will use [a public shared calendar from a service account in the Office 365 instance for NIST, as documented below](#service-account-with-published-nist-office-365-calendar). The team staff will add, change, and remove event invitations. Community members will view the web version or [iCalendar](https://en.wikipedia.org/wiki/ICalendar) files that are a standard that many calendar clients support.
```

Comment:
The sentence "Community members will view the web version or [iCalendar](https://en.wikipedia.org/wiki/ICalendar) files that are a standard that many calendar clients support." does not read well. Community members will have access to the .ics calendar invitation files in the web version or [iCalendar](https://en.wikipedia.org/wiki/ICalendar). These .ics files are standard a standard that many calendar clients support.